src/sentry/static/sentry/app/views/admin/adminLayout.tsx

```diff
@@ -1,64 +1,59 @@
 import React from 'react';
 import DocumentTitle from 'react-document-title';
+import {RouteComponentProps} from 'react-router';
 import styled from '@emotion/styled';
 
-import space from 'app/styles/space';
+import SettingsLayout from 'app/views/settings/components/settingsLayout';
 import SettingsNavigation from 'app/views/settings/components/settingsNavigation';
 
-const AdminLayout: React.FC = ({children}) => (
+const AdminNavigation = () => (
+  <SettingsNavigation
+    stickyTop="0"
+    navigationObjects={[
+      {
+        name: 'System Status',
+        items: [
+          {path: '/manage/', index: true, title: 'Overview'},
+          {path: '/manage/buffer/', title: 'Buffer'},
+          {path: '/manage/queue/', title: 'Queue'},
+          {path: '/manage/quotas/', title: 'Quotas'},
+          {path: '/manage/status/environment/', title: 'Environment'},
+          {path: '/manage/status/packages/', title: 'Packages'},
+          {path: '/manage/status/mail/', title: 'Mail'},
+          {path: '/manage/status/warnings/', title: 'Warnings'},
+          {path: '/manage/settings/', title: 'Settings'},
+        ],
+      },
+      {
+        name: 'Manage',
+        items: [
+          {path: '/manage/organizations/', title: 'Organizations'},
+          {path: '/manage/projects/', title: 'Projects'},
+          {path: '/manage/users/', title: 'Users'},
+        ],
+      },
+    ]}
+  />
+);
+
+type Props = {
+  children: React.ReactNode;
+} & RouteComponentProps<{}, {}>;
+
+const AdminLayout: React.FC<Props> = ({children, ...props}) => (
```

Comment:
should we remove `React.FC `as `children` is already part of `Props`? 
```suggestion
const AdminLayout = ({children, ...props}: Props) => (
```